include/binaryfusefilter.h

```diff
@@ -858,4 +860,91 @@ static inline bool binary_fuse8_deserialize(binary_fuse8_t * filter, const char
   return true;
 }
 
+// minimal bitfield implementation
+typedef uint8_t bitf_t;
+#define bitf_w (sizeof(bitf_t) * 8)
+#define bitf_sz(bits) (((bits) + bitf_w - 1) / bitf_w)
+#define bitf_word(bit) (bit / bitf_w)
+#define bitf_bit(bit) ((1U << (bit % bitf_w)) % 256)
+
```

Comment:
Same comment here, we should not have free standing macros as any project where 'ser' is a variable, function or type will break badly. So something like XOR_ser is needed (prefix so as to make the name clash unlikely).